pages/index.jsx

```diff
@@ -30,48 +32,51 @@ export default function HomePage() {
               <Stack.Item fill>
```

Comment:
Probably not introduced in this PR, but there a re a few deprecated Polaris components still in use:

<img width="514" alt="Screenshot 2023-04-27 at 10 13 26 AM" src="https://user-images.githubusercontent.com/690791/234889516-043c633c-f375-4006-b6ee-52178e0dac5a.png">

Not a blocker, but might be quick to update these in this PR. This repo doesn't see many changes, so it might be a good time.

Reply:
Actually, I just peeked at this, and it looks like most of the recommended components are still in Alpha (e.g. [Vertical Stack](https://polaris.shopify.com/components/layout-and-structure/vertical-stack)). Given that I don't work regularly with Polaris, and FED isn't my core competency, I will leave replacing deprecated Polaris components to someone more experienced.